backend/apps/owasp/management/commands/owasp_run_notification_worker.py

```
@@ -0,0 +1,113 @@
+        redis_conn = get_redis_connection("default")
+        stream_key = "owasp_notifications"
+        group_name = "notification_group"
+        consumer_name = "worker_1"
+
+        # Create consumer group if it doesn't exist
+        try:
+            redis_conn.xgroup_create(stream_key, group_name, id="0", mkstream=True)
+            print("Consumer group created successfully.")
+        except Exception:
+            # Group likely already exists
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)